app/main.py

```diff
@@ -1 +1,56 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str):
+        self.name = name
+
+    def learn_skill(self, skill: str):
+        self.__class__.skills.append(skill)
```

Comment:
Why do you use `__class__` attribute here?

Reply:
I thoght it was a class attribute and needed there __class__,  i was wrong(

Reply:
but I understood my mistake)